pkg/operator/controller/prometheus.go

```diff
@@ -221,6 +221,21 @@ func getAlertRules(runbookURLTemplate string) []promv1.Rule {
 				componentAlertLabelKey:    componentAlertLabelValue,
 			},
 		),
+		generateAlertRule(
+			"CDINoDefaultStorageClass",
+			"kubevirt_cdi_default_storageclasses == 0 and kubevirt_cdi_default_virt_storageclasses == 0 and kubevirt_cdi_datavolumes_pending_for_default_storageclass > 0",
```

Comment:
Maybe this should also just be about virt storage classes? DataImportCrons without explicit SC should all work if a virt storage class is defined

Reply:
Not sure I get you. User can define default and/or virt default SC, so why require the virt one specifically?

Reply:
Just a similar suggestion to https://github.com/kubevirt/containerized-data-importer/pull/2998#discussion_r1403587024. Reading it again though I think it's fine since it'll only alert if all of them are 0, so virt sc existence will bail us out of noise.